events/babylon_bulgarian_independence.txt

```diff
@@ -0,0 +1,72 @@
+namespace = babylon_bulgarian_independence
+babylon_bulgarian_independence.1 = {
+	type = country_event
+	placement = root
+	title = babylon_bulgarian_independence.1.t
+	desc = babylon_bulgarian_independence.1.d
+	flavor = babylon_bulgarian_independence.1.f
+	duration = 3
+	event_image = {
+		video = "gfx/event_pictures/unspecific_military_parade.bk2"
+	}
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "event:/SFX/Events/unspecific/military_parade"
+	icon = "gfx/interface/icons/event_icons/event_default.dds"
+	trigger = {
+	}
+	immediate = {
+		create_character = {
+			ruler = yes
+			culture = cu:bulgarian
+			interest_group = ig_armed_forces
+			traits = {
+			}
+		}
+		create_character = {
+			heir = yes
+			culture = cu:bulgarian
+			birth_date = 1836.1.1
```

Comment:
Is the birth_date necessary? If this pops late you may end up with an heir older than the ruler :s

Reply:
These were ported from emifas' branch, sorry. Same with the other bulgaria factor

Reply:
Sounds like we'll be having the same points to make when Emifas is done then, whoever grabs that pull request.

Reply:
So tag him, @Emifas

Reply:
Hi, if I make the heir have an undefined birth date he tends to be the same age as the auto-generated ruler. Stems from the issue with Bulgaria under the Ottomans being a monarchy and so the historical ruler at game start will generate an heir which also needs to be removed. I'm unsure if there is a way to kill the heir so i've used that as a workaround

Reply:
So why not make the heir older?

Reply:
There's also `disinherit_character = yes`